nodejs/awsx/ec2/internetGateway.ts

```diff
@@ -38,7 +39,7 @@ export class InternetGateway
             this.internetGateway = new aws.ec2.InternetGateway(name, {
                 ...args,
                 vpcId: vpc.vpc.id,
-            }, { parent: this });
+            }, parentOpts);

```

Comment:
this is stylistic.  But it matches what we do elsewhere, so i'd like it here for consistency.